### ISSUE TITLE
 interrupt/countershot.

### ISSUE THREAD

**Issue by cambrum** [Sidoine/Ovale#751]
hi, thanks for the updates they really help.
could fix the interrupts/countershot, please.
the 2 I have noticed are hunter and Sharman.

**Comment by ultijlam** [comment]
What is the problem you are seeing?

**Comment by cambrum** [comment]
hi, just not working at all

________________________________

**Comment by cambrum** [comment]
hi again, just logged on to my monk, and the interrupt is working. 
but still not working on hunter and shaman.
there was an update today I don't know if it was the interrupt fix.
hope you can help.

**Comment by cambrum** [comment]
hi, thanks for the update. the shaman interrupt is working now.
but the hunter interrupt still not working.

**Comment by orlith** [comment]
Same here. thx

**Comment by cambrum** [comment]
hi again, been on other characters, hunter/shadow priest/rogue/ death knight/demon hunter.
the interrupt is not on them.
I don't know if telling you this helps. 
hope you can help. thanks

**Comment by ultijlam** [comment]
I'm having trouble reproducing this. Can you check that the *Interrupt* checkbox that appears when you left-click on the Ovale icon is enabled?

**Comment by cambrum** [comment]
hi, the interrupt is working on mage/warrior/druid/shaman/monk.
but not on the others.
and as far as I can tell all boxes are ticked that should be ticked.
it all worked in battle for Azeroth and I haven't change anything,

________________________________

**Comment by orlith** [comment]
Also ticked here. Maybe it could be useful to know that I'm running wow in French. Maybe could be linked to interface language.

**Comment by cambrum** [comment]
hi again, the only part of the spell flash that doesn't work is the interrupt.
on mage/warrior/druid/shaman/monk. it all works .including interrupt.

**Comment by ultijlam** [comment]
I think I'm confused as to what's actually wrong. Is it that the interrupt spell is never suggested at all, or is it that the interrupt spell doesn't flash on your action bar?

**Comment by orlith** [comment]
For me it is never suggested with the latest version despite being ticked

**Comment by cambrum** [comment]
hi, the interrupt never suggested/flashes on my hunter.

________________________________

**Comment by ultijlam** [comment]
Do you know which was the last Ovale version where this was working correctly for you? I'm trying to narrow down what changes may have caused this issue.

**Comment by orlith** [comment]
I think it was the minus 2 version.
I'll try to test and get back to you

**Comment by cambrum** [comment]
hi, I don't know the version.
but it worked in battle of Azeroth.

________________________________

**Comment by orlith** [comment]
Test result : last working version suggesting interrupt for BM hunter is 9.0.26

**Comment by ultijlam** [comment]
Thanks for finding this information! This will help narrow down the changes to check. Can you confirm the current version (9.0.34) is also *not* working?

**Comment by orlith** [comment]
> Thanks for finding this information! This will help narrow down the changes to check. Can you confirm the current version (9.0.34) is also _not_ working?

Yes I confirm that this version still doesn't suggest interrupt.
Thx

**Comment by cambrum** [comment]
interrupt still not working.

________________________________

**Comment by chillaloha** [comment]
Interrupt indicator is still not working for either BM or MM Hunters. The icon lights up even when not in combat. When in combat it lights up on CD whether target is casting or not. This is a wonderful feature when working (as it did in BfA), but is not at all helpful now. This should light up when target is casting a dispel-able spell only.

**Comment by orlith** [comment]
Just for your info : using a custom script from my simulationcraft seems to fix the interrupt issue.

**Comment by chillaloha** [comment]
Not sure how to get your custom simulationcraft script - is this in Git?

**Comment by orlith** [comment]
If you want to check here is my simc 
[simc_gui.simc.txt](https://github.com/Sidoine/Ovale/files/6276531/simc_gui.simc.txt)
